Update Brazilian Translation PT-BR

williansti opened the pull request:
I updated the Brazilian Portuguese translation, I added the texts that were not translated.

GreemDev commented:
You added translations to pt_BR that were identical to the English version, please remove them.
Looking through this PR I notice that other languages have this problem but aren't triggering the build to fail. I'll clean those up after this is merged

williansti commented:
I suspect it was line 190 where I added "program" but then changed it to "software" which is the same in English, well I removed it, let's see if it works now.
